.github/workflows/micropython_lvgl_ci.yml

```diff
@@ -83,12 +83,14 @@ jobs:
         make -j $(nproc) -C ports/rp2 BOARD=RPI_PICO USER_C_MODULES=../../user_modules/lv_binding_micropython/micropython.cmake
 
     # ESP32 port
+        # USER_C_MODULES defined in mpconfigboard.cmake
```

Comment:
Please remove this line, it is incorrect.
In ESP32 port `USER_C_MODULES` is defined in `esp32_common.cmake` file:
https://github.com/lvgl/lv_micropython/blob/master/ports/esp32/esp32_common.cmake#L14